### PR TITLE
Adding extension.ts tests

### PR DESCRIPTION
**Pull request by jamarw**
[AB#233670](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/233670)

### CODE REVIEW

File: docs-markdown/src/test/suite/extension.test.ts [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
 
+suite('Extension tests', () => {
+	test('installedExtensionsCheck', () => {
+		const spy = chai.spy(installedExtensionsCheck);
 		installedExtensionsCheck();
-
-		assert.equal(-1, [1, 2, 3].indexOf(5));
-		assert.equal(-1, [1, 2, 3].indexOf(0));
+		sleep(sleepTime);
+		expect(spy).to.be.spy;
```
Comment (bharney0) [inline docs-markdown/src/test/suite/extension.test.ts:14]: Im not sure I understand what is being tested here. It looks like we are checking if the function you setup as a spy is a spy? Maybe we want to check if the function has been called and call activate?

File: docs-markdown/src/test/suite/extension.test.ts [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
+	});
+	test('setupAutoComplete', () => {
+		const spy = chai.spy(setupAutoComplete);
+		setupAutoComplete();
+		sleep(sleepTime);
+		expect(spy).to.be.spy;
```
Comment (bharney0) [inline docs-markdown/src/test/suite/extension.test.ts:20]: same here

File: docs-markdown/src/test/suite/extension.test.ts [hunk 3]
```diff
@@ -1,19 +1,28 @@
...
+	});
+	test('deactivate', () => {
+		const spy = chai.spy(deactivate);
+		deactivate();
+		sleep(sleepTime);
+		expect(spy).to.be.spy;
```
Comment (bharney0) [inline docs-markdown/src/test/suite/extension.test.ts:26]: and here

File: docs-markdown/src/test/suite/extension.test.ts (hunk 1 above)
Comment (jamarw) [reply docs-markdown/src/test/suite/extension.test.ts:14]: Hi @bharney0 , I'll take a look.  It should test for calls but maybe I committed this when I was testing for external extensions.

Comment (jamarw) [reply docs-markdown/src/test/suite/extension.test.ts]: Hi @bharney0 , forgot to push my last commit.  Can you take a look now?  Thanks!

**Pull request merged**